Group dependency updates

colincasey opened the pull request:
Adding configuration to dependabot to take advantage of [group updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups).

[GUS-W-13579117](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-13579117)

colincasey committed:
Group dependency updates

Adding configuration to dependabot to take advantage of [group updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups).

[GUS-W-13579117](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-13579117)

edmorley commented on .github/dependabot.yml:
```diff
@@ -4,3 +4,8 @@ updates:
     schedule:
       interval: "monthly"
+    groups:
+      dependencies:
```
Same comment as https://github.com/forcedotcom/sf-fx-runtime-java/pull/439/files#r1388586063 :-)

colincasey replied:
added GH action updates and renamed this group to `java-dependencies`

colincasey committed:
Group dependency updates

Adding configuration to dependabot to take advantage of [group updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups).

[GUS-W-13579117](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-13579117)
